pkg/specgen/namespaces.go

```diff
@@ -301,3 +306,20 @@ func ParseNetworkNamespace(ns string, rootlessDefaultCNI bool) (Namespace, []str
 
 	return toReturn, cniNetworks, nil
 }
+
+func ParseNetworkString(network string) (Namespace, []string, map[string][]string, error) {
+	var networkOptions map[string][]string
+	parts := strings.SplitN(network, ":", 2)
+
+	ns, cniNets, err := ParseNetworkNamespace(network, containerConfig.Containers.RootlessNetworking == "cni")
```

Comment:
Does this compile? I don't think we have `containerConfig` as a variable in pkg/specgen?

Reply:
Yes.  I will check that.

./pod_validate.go:	containerConfig = util.DefaultContainerConfig()